File: BoardGames/HitOrMiss/include/libpython/score_view.py
import score_model

def build_score_lines( score ) :
  try :
    lines  = list()
    games  = score_model.get_games  ( score )
    hits   = score_model.get_hits   ( score )
    misses = score_model.get_misses ( score )

    lines.append( 'Game Results:' )
    lines.append( '\tGames Played......... {0:2d}'         .format( games  ) )
    lines.append( '\tPlayer has won....... {0:2d} time(s)' .format( hits   ) )
    lines.append( '\tPlayer has miss...... {0:2d} time(s)' .format( misses ) )

    return lines
  except KeyError :
    return None

File: BoardGames/HitOrMiss/include/libruby/score_view.rb
require_relative 'score_model'

def build_score_lines( score )
  begin
    lines  = Array.new
    games  = get_games  ( score )
    hits   = get_hits   ( score )
    misses = get_misses ( score )

    lines << 'Game Results:'
    lines << sprintf( "\tGames Played......... %2d"        , games  )
    lines << sprintf( "\tPlayer has won....... %2d time(s)", hits   )
    lines << sprintf( "\tPlayer has miss...... %2d time(s)", misses )

    return lines
  rescue KeyError
    nil
  end
end
